thrall/app/lib/kinesis/ThrallEventConsumer.scala

```diff
@@ -65,13 +66,14 @@ class ThrallEventConsumer(es: ElasticSearch,
     }
   }
 
-  private def parseRecord(r: Record):Option[UpdateMessage] = {
+  def parseRecord(r: Record):Option[UpdateMessage] = parseRecord(r.getData.array, r.getApproximateArrivalTimestamp.toInstant)
+
+  def parseRecord(r: Array[Byte], timestamp: Instant):Option[UpdateMessage] = {
```

Comment:
Is there a reason this is overloaded in this way? I'm reading this as sugar for two val assignments at the start of the second declaration of `parseRecord`, perhaps to clarify the way the `Record` is used in the function?

Reply:
This was overloaded as I had originally preserved the code of both ways (direct KCL and KCL via Akka streams) of consuming the Kinesis stream and they produced different types. As such one was a shim for the other without muddying the calling code.

However I removed the dual code towards the end of my work but didn't clean up the methods in this class.